change/@ni-nimble-angular-ed76a9f1-e9fd-4735-90e4-64ef28831872.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@m-akinc disabling automerge as main has a pending build (the latest commit is not applying package updates)
Please make sure to only autocomplete when main is ready to merge. i.e. you can see on main that a new commit will soon be pushed so the autocomplete will not succeed as-is anyway. [See discussion](https://teams.microsoft.com/l/message/19:02dd02acd4ef470889e8e2afefd90b02@thread.tacv2/1715283494960?tenantId=87ba1f9a-44cd-43a6-b008-6fdb45a5204e&groupId=9ee065d7-3898-4245-82f6-76e86084b8b1&parentMessageId=1714135739681&teamName=Design%20System&channelName=Dev%20Squad&createdTime=1715283494960)